packages/wallets/src/features/cfd/modals/VerificationFailedModal/VerificationFailedModal.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { FC } from 'react';
+import { ModalWrapper } from '../../../../components/Base';
+import { VerificationFailed } from '../../screens';
+
+const VerificationFailedModal: FC<unknown> = () => {
```

Comment:
Why is it unknown ?

Reply:
There are no props passed, usually I used `unknown` to specify it